moon-store-mac-os/moon-store-mac-os/Views/Product/ProductMockData.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  ProductMockData.swift
+//  moon-store-mac-os
+//
+//  Created by Diana Zeledon on 13/12/24.
+//
+
+import Foundation
+import SwiftUI
+
+struct ProductoMockData: Identifiable {
+    let id = UUID()
+    let name: String
+    let category: CategoryType
+    let inStock: Int
+    let price: String
+
+}
+
+enum CategoryType: String, CaseIterable {
+    case frutas = "Frutas"
+    case lacteos = "Lácteos"
+    case panaderia = "Panadería"
+    case verduras = "Verduras"
+    case otros = "Otros"
+    
+    // Color asociado a cada categoría
+    var color: Color {
+        switch self {
+        case .frutas: return .green
+        case .lacteos: return .blue
+        case .panaderia: return .orange
+        case .verduras: return .red
+        case .otros: return .gray
```

Comment:
You could use implicit return.
```suggestion
        case .frutas: .green
        case .lacteos: .blue
        case .panaderia: .orange
        case .verduras: .red
        case .otros: .gray
```